website/scripts/website-build.sh

```diff
@@ -7,22 +7,27 @@ CLONE_DIR=website-preview
 from_cache=false
 
 if [ -d "$PREVIEW_DIR" ]; then
-  echo "$PREVIEW_DIR found"
-  CLONE_DIR="$PREVIEW_DIR-tmp"
-  from_cache=true
+    echo "$PREVIEW_DIR found"
+    CLONE_DIR="$PREVIEW_DIR-tmp"
+    from_cache=true
 fi
 
 # Clone the terraform-website project, if needed
 echo "⏳ Cloning the terraform-website repo, this might take a while..."
-git clone --depth=1 --branch=master https://github.com/hashicorp/$REPO_TO_CLONE.git "$CLONE_DIR"
+git clone --depth=1 --branch=kevin/terraform-split https://github.com/hashicorp/$REPO_TO_CLONE.git "$CLONE_DIR"
```

Comment:
Apply this change when ready to merge

```suggestion
git clone --depth=1 --branch=master https://github.com/hashicorp/$REPO_TO_CLONE.git "$CLONE_DIR"
```